pennylane/drawer/tape_mpl.py

```diff
@@ -131,15 +131,14 @@ def _tape_mpl(tape, wire_order=None, show_all_wires=False, decimals=None, **kwar
                 specialfunc(drawer, layer, mapped_wires, op)
 
             else:
-                op_control_wires = getattr(op, "control_wires", [])
+                op_control_wires, control_values = unwrap_controls(op)
```

Comment:
The bug affected the mpl drawer output as well. I have added the fix here and visually inspected a couple cases and it does seem to work. However, I don't know enough about mpl to know how to test this properly. @albi3ro you wrote the mpl drawer right, any advice here? (Does this actually need its own tests if it works now for the underlying tapes?)

Reply:
@glassnotes  I can write up a quick test for this, since I'm already familiar with how to test matplotlib code.  OK if I just push it to your branch?

Reply:
Actually you should be able to just re-use code from `test_tape_mpl.py:TestControlledGates:test_control_values_bool`.

You can either copy or modify that test to have a nested controlled operation, instead of a single level of control with four control wires.

Reply:
And if you want to know how it works, there is documentation at `doc/developement/guide/tests.rst`

Reply:
@albi3ro thanks! Please feel free to push to the branch if it is something you can concoct quickly :slightly_smiling_face: (otherwise I can give it a go tomorrow!)